src/targets/services/helpers.js

```diff
@@ -0,0 +1,71 @@
+import last from 'lodash/last'
+import get from 'lodash/get'
+
+/**
+ * Returns the changes or all documents if it's more efficient
```

Comment:
What does efficient mean here ? Based on what ?

Reply:
I think this could be moved to cozy-doctypes right ? Maybe not in this PR but a TODO could be good.

Reply:
I updated the docstring.

Reply:
Thanks for updating the docstring. I wonder if it could be moved to cozy-stack-client::DocumentCollection, do you think it would have its place there ?

Reply:
It definitely has its place there. I'm going to open a PR.